docs/sources/live/configure-grafana-live.md

```diff
@@ -0,0 +1,107 @@
++++
+title = "Configure Grafana Live"
+description = "Grafana Live configuration guide"
+keywords = ["Grafana", "live", "guide", "websocket"]
+weight = 120
++++
+
+# Configure Grafana Live
+
+Grafana Live is enabled by default. As of Grafana v8.0, it has a strict default for a maximum number of connections per Grafana server instance. 
+
+## Max number of connections
+
+Grafana Live uses persistent connections (WebSocket at the moment) to deliver real-time updates to clients. You might need to tune your server infrastructure a bit.
+
+The number of maximum connections users can establish with Grafana is limited to 100 by default. See [max_connections]({{< relref "../administration/configuration.md#max_connections" >}}) option.
+
+In case you want to increase this limit, ensure that your server and infrastructure allow handling more connections. The following sections discuss several common problems which could happen when managing persistent connections, in particular WebSocket connections.
+
+### WebSocket
+
+Grafana Live uses WebSocket protocol to deliver real-time updates to a frontend application. WebSocket is a persistent connection that starts with an HTTP Upgrade request (using the same HTTP port as the rest of Grafana) and then switches to a TCP mode where WebSocket frames can travel in both directions between a client and a server.
+
+Each logged-in user opens a WebSocket connection – one per browser tab.
+
+Introducing a persistent connection leads to some things Grafana users should be aware of.
```

Comment:
```suggestion
Introducing a persistent connection can lead to certain situations that the Grafana users should be aware of.
```

Reply:
Should we specify the situations and give a few examples?

Reply:
Rewrote this part in another way